node-app/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "node-app.fullname" . }}-role-binding
+  labels:
+    {{- include "node-app.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "node-app.fullname" . }}-role
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "node-app.serviceAccountName" . }}
```

Comment:
this should be changed to use the init service account once that is extracted from an init container to a job

Reply:
i set this just to see if the car generation code works